File: Sistema/API/app/application/Model/FormatoModel.py
from .BaseModel import BaseModel
from .AudioModel import AudioSchema
from .TecnologiaProyeccionModel import TecnologiaProyeccionSchema
from ..Shared import db
from ..Shared import ma

class Formato(BaseModel):
    __tablename__ = 'formato'
    nombre = db.Column(db.String(128), nullable=False)
    id_audio = db.Column(db.Integer, db.ForeignKey('audio.id'))
    audio = db.relationship("Audio", backref=db.backref("audio", uselist=False),lazy='subquery')
    id_tecnologia_proyeccion = db.Column(db.Integer, db.ForeignKey('tecnologia_proyeccion.id'))
    tecnologiaProyeccion = db.relationship("TecnologiaProyeccion", backref=db.backref("tecnologia_proyeccion", uselist=False),lazy='subquery')

class FormatoSchema(ma.SQLAlchemyAutoSchema):
    class Meta:
        model = Formato
        load_instance = True
        sqla_session = db.session
    audio = ma.Nested(AudioSchema())
    tecnologiaProyeccion = ma.Nested(TecnologiaProyeccionSchema())

File: Sistema/FE/app/src/app/entities/formato/formato.model.ts
import { IAudio } from "../audio/audio.model";
import { ITecnologiaProyeccion } from "../tecnologia-proyeccion/tecnologia-proyeccion.model";

export interface IFormato {
    id?: number;
    nombre?: string;
    tecnologiaProyeccion?:ITecnologiaProyeccion;
    audio?:IAudio;
}

export class Formato implements IFormato {
    constructor(
        public id?: number,
        public nombre?: string,
        public tecnologiaProyeccion?:ITecnologiaProyeccion,
        public audio?:IAudio
    ) {

    }
}